torch/_inductor/cpp_builder.py

```diff
@@ -1006,11 +1006,11 @@ class CppBuilder:
             3. Final target file: output_dir/name.ext
     """
 
-    def get_shared_lib_ext(self) -> str:
-        SHARED_LIB_EXT = ".dll" if _IS_WINDOWS else ".so"
+    def __get_python_module_ext(self) -> str:
```

Comment:
nit: double underscore prefix seems not common in python? Use single underscore if it's internal?

Reply:
> nit: double underscore prefix seems not common in python? Use single underscore if it's internal?

I'm mark these functions as class private functions, and avoid to user call them.